### ISSUE TITLE
code run

### ISSUE THREAD

**Issue by oceanphy** [Sh-Fang/TCMatch#1]
hello, can this code run on the Windows platform?

**Comment by Sh-Fang** [comment]
Yes, this code can run on the Windows platform. To build and run it, you will need to install CMake version 3.16 or higher.

**Comment by oceanphy** [comment]
> Yes, this code can run on the Windows platform. To build and run it, you will need to install CMake version 3.16 or higher.

but, i catch a wrong,
![图片](https://github.com/user-attachments/assets/7fbbb158-c477-4f29-b429-da22ac30a10d)
this si my cmake version
![图片](https://github.com/user-attachments/assets/6560572f-20bb-401d-94cd-6f16fe5e717c)
